site/docs/v1/tech/events-webhooks/events.adoc

```diff
@@ -64,10 +62,14 @@ A temporal action is one that has a start time and and a duration, when a phase
 user.action
 ----
 
+=== Event Scope
+
+include::docs/v1/tech/events-webhooks/_tenant-or-application-scoped-event.adoc[]
+
 === Event Body
 
 [.api]
-[field]#event.action# [type]#[String]#::
+[field]#event.action# [type]#[String]::
```

Comment:
Typo? Looks like you didn't mean to remove the trailing `#`

Reply:
Correct, thanks.